fuzz/fuzz_targets/fuzzed_ops.rs

```diff
@@ -0,0 +1,166 @@
+use fuel_vm::prelude::{Opcode, RegisterId, Immediate12, Immediate18, Immediate24};
```

Comment:
I think that deriving `arbitrary::Arbitrary` in https://github.com/FuelLabs/fuel-asm/blob/master/src/opcode.rs would eliminate the need for this file